src/userInterface/panels/CommandPanel/CommandPanel.tsx

```diff
@@ -24,43 +38,73 @@ export const CommandPanel: React.FC<{
     }
 
     const onSaveHandler = () => {
-        if (confirm("Do you want to cretate new Json Config?")) {
-            props.commandEvent(CommandEvent.saveAs)
-                .then((response: any) => {
-                    const createdId = response.data.data.items[0].id;
-                    props.reloadJsonConfigs(createdId);
-                    alert("Data saved successfully!");
-                })
-                .catch((error: any) => {
-                    JsonConfigCommandCenter.errorAlert("Save Cancelled!", error);
-                });
+        if (!isValidFileName()) {
+            message.error(`${LOCALIZATION.SAVE_INVALID_FILE}`);
+        }
+        else {
+            confirm({
+                title: `${LOCALIZATION.SAVE_TITLE}`,
+                icon: <ExclamationCircleOutlined />,
+                content: `${LOCALIZATION.SAVE_CONTENT}`,
+                onOk() {
+                    props.commandEvent(CommandEvent.saveAs)
+                        .then((response: any) => {
+                            const createdId = response.data.data.items[0].id;
+                            props.reloadJsonConfigs(createdId);
+                            message.success(`${LOCALIZATION.SAVE_SUCCESS}`);
+                        })
+                        .catch((error: any) => {
+                            message.error(`${LOCALIZATION.SAVE_ERROR}`.replace('{{error}}', `${extractErrorMessage(error)}`));
+                        });
+                }
+            });
         }
     }
 
     const onUpdateHandler = () => {
-        if (confirm("Do you want to update file with new changes?")) {
-            props.commandEvent(CommandEvent.update)
-                .then((response: any) => {
-                    const createdId = response.data.data.items[0].id;
-                    props.reloadJsonConfigs(createdId);
-                    alert("Data updated successfully!");
-                })
-                .catch((error: any) => {
-                    JsonConfigCommandCenter.errorAlert("Update failed!", error);
-                });
+        if (!isValidFileName()) {
+            message.error(`${LOCALIZATION.UPLOAD_INVALID_FILE}`);
+        }
+        else {
+            confirm({
+                title: `${LOCALIZATION.UPLOAD_TITLE}`,
+                icon: <ExclamationCircleOutlined />,
+                content: `${LOCALIZATION.UPLOAD_CONTENT}`,
+                onOk() {
+                    props.commandEvent(CommandEvent.update)
+                        .then((response: any) => {
+                            const createdId = response.data.data.items[0].id;
+                            props.reloadJsonConfigs(createdId);
+                            message.success(`${LOCALIZATION.UPLOAD_SUCCESS}`);
```

Comment:
can't you put this as message.success(LOCALIZATION.UPLOAD_SUCCESS);

Reply:
yup. fixed it